### DISCUSSION TITLE
CUE Team at KubeConEU 2023- Amsterdam 18-21

### DISCUSSION THREAD

**Discussion by Lyoness** [cue-lang/cue#2340] [Announcements]
## Calling all CUE community members headed to KubeCon EU 2023:

@rogpeppe @mpvl @myitcv @Lyoness and @dominikdm will be headed to Amsterdam next week from Tuesday through Friday for [KubeCon EU](https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/). 

If you plan to be there as an attendee or are otherwise in the area and would like to meet to talk about CUE, please comment in the thread or email carmen@cue.works.  

We'll come with some fun CUE swag.

**Comment by mxey** [comment]
I'll be there and would like to pick up some swag and talk about the modules proposal :)

**Reply by myitcv** [reply]
Thanks for replying here! Look forward to chatting. I've replied on Slack where we can coordinate timings etc via DM.
